boards/cc2538dk/Makefile.include

```diff
@@ -7,7 +7,7 @@ PROGRAMMER_SERIAL ?= 06EB
 
 # setup serial terminal
 # the debug UART is always the second tty with the matching serial number:
-PORT_LINUX ?= $(word 2,$(shell $(RIOTTOOLS)/usb-serial/find-tty.sh '^$(PROGRAMMER_SERIAL)'))
+PORT_LINUX ?= $(word 1,$(shell $(RIOTTOOLS)/usb-serial/find-tty.sh '^$(PROGRAMMER_SERIAL)'))
```

Comment:
I tested this on my setup and it works fine. There might be a problem with how this is defined but lets not mix things. If there is a problem we should open another PR about it, and figure out why it works for me (and presumably others) but not you.